### PR TITLE
OBPIH-3084 Dashboard: Value of inventory

### PR DESCRIPTION
**Pull request by bamoure**

### CODE REVIEW

File: src/js/components/tablero/NumberCard.jsx [hunk 1]
```diff
@@ -103,7 +104,7 @@ const NumberCard = SortableElement(({
               translate(cardTitle.code, cardTitle.message)
              : cardTitle}
           </span>
-          <span className="result-card"> {cardNumber.toLocaleString()} </span>
+          <span className="result-card"> {cardNumberType === 'number' ? cardNumber.toLocaleString() : `${cardNumber.toLocaleString()}USD`} </span>
```
Comment (awalkowiak) [inline src/js/components/tablero/NumberCard.jsx:107]: Should not this support other currencies?

Comment (bamoure) [reply src/js/components/tablero/NumberCard.jsx:107]: The ticket specified USD. Should we implement it for other currencies?

Comment (jmiranda) [reply src/js/components/tablero/NumberCard.jsx:107]: @bamoure Yeah we should pull the default currency from the configuration. Let me know if you need help with that.

Comment (jmiranda) [reply src/js/components/tablero/NumberCard.jsx:107]: @awalkowiak Can you help add the default currency to the config endpoint?

**Review by jmiranda — CHANGES_REQUESTED** [review]

File: src/js/components/tablero/NumberCard.jsx (hunk 1 above)
Comment (awalkowiak) [reply src/js/components/tablero/NumberCard.jsx:107]: Sure I'll look into that today

Comment (awalkowiak) [reply src/js/components/tablero/NumberCard.jsx]: @jmiranda done

**Review by awalkowiak — APPROVED** [review]

**Pull request merged**